### DISCUSSION TITLE
Number of decision vectors

### DISCUSSION THREAD

**Discussion by dietmarwo** [esa/SpOC3#10] [Challenge 1: Torso Decompositions]
At https://optimise.esa.int/challenge/spoc-3-torso-decompositions/About we have:
"Your solution is allowed to contain up to 20 decision vectors" 
Are you sure all uploaded solutions shown at the leaderboard follow this rule?
Do you plan to delete those which violate this rule before the competition ends?
Or has the valid number "secretly" changed and we have a new limit?

This is not a new issue, we had this already at SpOC2: https://github.com/esa/SpOC2/discussions/23

**Comment by maxbannach** [comment]
Of course, the upper bound was not "secretly changed." But indeed, the medium graph was wrongly configured in the backend. Luckily, only one user exploited this bug. We deleted the corresponding submissions.

Thanks for spotting it! :)

**Comment by dietmarwo** [comment]
Thks for fixing the issue. This one user was me, because of https://github.com/esa/SpOC2/discussions/23 in SpOC2 I was testing the limits. It was never my intention to "exploit" anything, I just wanted to make sure that the limits are checked and reported the issue immediately.
